app/src/main/java/com/x8bit/bitwarden/data/vault/repository/VaultRepositoryImpl.kt

```diff
@@ -553,6 +553,24 @@ class VaultRepositoryImpl(
         mutableTotpCodeResultFlow.tryEmit(totpCodeResult)
     }
 
+    override suspend fun unlockVaultWithAuthenticatorSyncKey(): VaultUnlockResult {
+        val userId = activeUserId ?: return VaultUnlockResult.InvalidStateError
```

Comment:
Is this going to work as something that is based on the current user? We'll need to unlock the vaults for all users that have enabled the feature. Also, I'm not sure it should necessarily be an error if the current user does not have a key for that same reason...it's a valid state to not have a key.

Reply:
We spoke about this a bit offline, too. 

TLDR; we want this to be per user, and the branch is updated as such.